includes/features.php

```diff
@@ -120,6 +120,7 @@ function today_display_feature_horizontal( $post, $args=array() ) {
 	if ( ! $post instanceof WP_Post ) return;
 
 	$type          = isset( $args['layout__type'] ) ? $args['layout__type'] : 'secondary';
+	$excerpt_len   = isset( $args['excerpt_length'] ) ? $args['excerpt_length'] : null;
```

Comment:
Maybe in a future PR, it'd be nice if `excerpt_length` were also registered as a new `[post-list]` shortcode param for consistency with all the other available options in these functions.  See `today_post_list_sc_atts()`